scope/scope.go

```diff
@@ -338,7 +338,8 @@ func UseLogger(logger telemetry.Logger) {
 	lock.Lock()
 	defer lock.Unlock()
 
-	if defaultLogger != nil {
+	// Skip cloning when defaultLogger is uninitialized or the given logger is the same instance.
+	if defaultLogger != nil && logger == defaultLogger {
```

Comment:
Since I updated this file, does that means we need to release another version of telemetry? Should this PR'd in a separate PR?

Reply:
https://github.com/tetratelabs/telemetry/pull/14

Reply:
This change is wrong. I'll update the test shortly.